docs/cli-flags.md

```diff
@@ -11,3 +11,4 @@ Here are listed all the available flags, their default values and the accepted o
 | The path of the dotenv file to load during launch | `-e` | `--env-path` |  |
 | Expose Prometheus metrics |  | `--expose-metrics` | `true`  |
 | Enable tracing            |  | `--enable-tracing` | `false` |
+| Custom log level 35 | `-c`  | `--custom-log-level` | `audit` |
```

Comment:
why 35? The user should be able to set the levelName;levelValue in the same list

e.g.

```
--custom-log-level=audit:35,wow:11,success:70
```